test_runner/performance/test_storage_controller_scale.py

```diff
@@ -59,6 +130,9 @@ def test_storage_controller_many_tenants(
         ]
     )
 
+    # TODO: default tenant does not have a secondary location
+    env.storage_controller.allowed_errors.append(".*Scheduling error when draining pageserver.*")
```

Comment:
I can't think of a good reason that I left the default tenant in this test: we should probably just avoid creating it

Reply:
[622e255](https://github.com/neondatabase/neon/pull/8173/commits/622e255ffe4d6748d09cec0b0c83381daf1afc4d)